src/views/v2/month/mobile-events/mobile-day/day-marker.php

```diff
@@ -14,12 +14,13 @@
  * @var string $day_date Date for this marker, following `Y-m-d` format.
  *
  */
+use Tribe__Date_Utils as Date;
```

Comment:
For consistency with what we do in other code fragments, and to avoid confusion, change the alias to `Dates` (plural).